src/Parser/Priest/Shadow/Modules/Spells/VoidformsTab.js

```diff
@@ -10,7 +10,7 @@ const VoidformsTab = ({ voidforms = [], insanityEvents, ...modules }) => {
         key={i}
         {...voidform}
         {...modules}
-        insanityEvents={insanityEvents.filter(event => event.timestamp >= voidform.start && event.timestamp <= voidform.ended)}
+        insanityEvents={insanityEvents.filter(event => event.timestamp >= voidform.start && event.timestamp <= voidform.start + voidform.duration)}
```

Comment:
I recommend using parenthesis when combining operators to make it obvious that it gets executed right. Does it do `(event.timestamp <= voidform.start) + voidform.duration` or `event.timestamp <= (voidform.start + voidform.duration)`?